frontend/src/Components/Forms/AddPlantForm.js

```diff
@@ -11,9 +11,8 @@ export default function AddPlantForm({ savePlantData }) {
 
   const history = useHistory()
   const initialInput = {
-    id: '',
-    name: '',
-    species: '',
+    plantname: '',
+    plantspecies: '',
```

Comment:
Wouldn't it have been simpler to name the object more meaningful? (initialPlant or so)